### ISSUE TITLE
More clarity for docs.

### ISSUE THREAD

**Issue by SiddharthShyniben** [JSJitsu/ai-battle-website#122]
Hi, I would like to know a few more things about the game:
- What happens when a hero with maximum health moves into a heath well?
- What is the maximum health in the first place? Is is 100? Or is there a limit?
Thanks in advance!

**Comment by Asuza** [comment]
I believe a hero with maximum health will not heal any more than that, with a maximum health of 100. I should add that information to the game rules, thanks for bringing it up.

**Comment by SiddharthShyniben** [comment]
Thank you!

There is a lot of missing information I think. If you want I can point them out.

**Comment by Asuza** [comment]
Absolutely, @SiddharthShyniben. I don't often get feedback, so feel free to point out anything.

**Comment by SiddharthShyniben** [comment]
Alright, I'll just reread the docs just in case I missed anything.

**Comment by SiddharthShyniben** [comment]
Here's a few:

> Diamond mines will be denoted by "DXX" where the Xs are numbers. 

What numbers? The number of gems in the mine or what?

And there is no description of what actually is contained in the gameData object.

**Comment by SiddharthShyniben** [comment]
And I also see trees on the [JSFight](https://JSFight.club) website. What are they? Obstacles? Or is it possible to walk through them?

**Comment by Asuza** [comment]
Those are obstacles, currently. There's no way to walk through.

**Comment by SiddharthShyniben** [comment]
> Those are obstacles, currently. There's no way to walk through.

But I don't see any mention of them.

**Comment by Asuza** [comment]
@SiddharthShyniben, did you have anything you definitely want to see added to the docs? Or were you able to figure everything out?
